### DISCUSSION TITLE
Unable to contact sieve

### DISCUSSION THREAD

**Discussion by mmbourgc** [Mailu/Mailu#3629] [User support]
Hello, i've installed mailu with helm, version 2.1.1. I'm using roundcube as my webmail. Everything's fine except sieve filters. 

When i try to display the filters, there's a error in roundcube logs : 

`*184239 FastCGI sent in stderr: "PHP message: PHP Warning:  stream_socket_client(): Unable to connect to mailu-front.mail.svc.cluster.local:14190 (Connection refused) in /var/www/roundcube/vendor/pear/net_socket/Net/Socket.php on line 158" while reading response header from upstream, client: 10.244.0.203, server: , request: "GET /?_task=settings&_action=plugin.managesieve HTTP/1.1", upstream: "fastcgi://unix:/var/run/php8-fpm.sock:", host: "xxxxxxxxxxxxx.com", referrer: "https://xxxxxxxxxxxxx.com/webmail/?_task=settings&_action=responses"
`

I've checked mailu-front service port 14190 with telnet and i get a Connection refused.

I've cheked mailu-dovecot service port 4190 with telnet and it's ok.

I've manage to make it work by modifying roundcube conf ( /var/www/roundcube/config/config.inc.php in the pod) to point managesieve_host parameter to dovecot pot on port 4190

I've checked the chart values but i don't know what could be wrong. Does anyone have any idea what might be misconfigured ?

By the way, nice job for everything :)

**Comment by nextgens** [comment]
Front should be listening on port 14190 and reachable from webmail. If it isn't there is a problem with the helm chart (network policy maybe?); file a bug on their repository please.

**Reply by mmbourgc** [reply]
Ok, thanks
